sdcm/utils/raft/__init__.py

```diff
@@ -51,14 +51,11 @@ class MessageTimeout(NamedTuple):
 
 ABORT_DECOMMISSION_LOG_PATTERNS: Iterable[MessagePosition] = [
     MessagePosition("api - decommission", LogPosition.BEGIN),
-    MessagePosition("DECOMMISSIONING: unbootstrap starts", LogPosition.BEGIN),
-    MessagePosition("DECOMMISSIONING: unbootstrap done", LogPosition.END),
-    MessagePosition("becoming a group 0 non-voter", LogPosition.END),
-    MessagePosition("became a group 0 non-voter", LogPosition.END),
-    MessagePosition("leaving token ring", LogPosition.END),
-    MessagePosition("left token ring", LogPosition.END),
     MessagePosition("raft_topology - decommission: waiting for completion", LogPosition.BEGIN),
-    MessagePosition("repair - decommission_with_repair", LogPosition.END)
+    MessagePosition("repair - decommission_with_repair", LogPosition.END),
+    MessagePosition("raft_topology - request decommission for", LogPosition.BEGIN),
+    MessagePosition("storage_service - Started batchlog replay for decommission", LogPosition.BEGIN),
+    MessagePosition("storage_service - Finished batchlog replay for decommission", LogPosition.BEGIN),
```

Comment:
This one `MessagePosition("storage_service - Finished batchlog replay for decommission", LogPosition.BEGIN),` i think not needed